package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "oyster-streamable",
-  "version": "2.1.5",
+  "version": "2.2.1",
```

Comment:
why not 2.1.6 or 2.2.0

Reply:
Its a minor update, and I already published to 2.2.0 to QA